js/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "inji-vci-client",
+  "version": "1.0.0",
+  "description": "Contains OpenId4VCI specification related source code and documentation",
+  "main": "src/index.js",
+  "type": "commonjs",
+  "scripts": {
+    "test": "jest --coverage",
+    "verify": "npm test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mosip/inji-vci-client.git"
+  },
+  "keywords": [
+    "OpenId4VCI",
+    "vci-client"
+  ],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mosip/inji-vci-client/issues"
+  },
+  "homepage": "https://github.com/mosip/inji-vci-client#readme",
+  "dependencies": {
+    "asn1js": "^3.0.5",
+    "axios": "^1.7.2",
+    "buffer": "^6.0.3",
+    "forge": "^2.3.0",
+    "js-base64": "^3.7.7",
+    "jwt-decode": "^4.0.0",
+    "node-fetch": "^3.3.2",
+    "node-forge": "^1.3.1",
+    "node-jose": "^2.2.0",
+    "pem-jwk": "^2.0.0",
+    "simple-pem2jwk": "^0.2.4",
+    "sinon": "^18.0.0"
```

Comment:
Please remove unused dependencies and move any dependencies related to testing to devDependencies